run_final.sh

```diff
@@ -0,0 +1,22 @@
+# Define arrays for threads, warps, and matrix sizes
+matrix_sizes=(16 32 64 128 256 512)
+tcsizes=(8 16 32)
+tcnums=(4 8 16 32)
+#lsulanes=(4 16)
+#cores=(32)
+
+
+# Loop through each combination of threads and warps
```

Comment:
```suggestion
# Loop through each combination of matrix_size, tcsize and tcnum.
```
The threads and warps seem to be unchanged in every iteration.

Reply:
Fixed.